crates/meta/src/types.rs

```diff
@@ -202,18 +205,42 @@ impl BqlType {
             decimal_item if decimal_item.starts_with(b"Decimal") => {
                 Self::_decimal_type(decimal_item)
             },
+            fixed_string_item if fixed_string_item.starts_with(b"FixedString") => {
+                match &fixed_string_item[b"FixedString".len()..] {
+                    [b'(', len @ .., b')'] => {
+                        Ok(BqlType::FixedString(Self::_parse_num(len)?))
+                    }
+                    _ => Err(MetaError::UnknownBqlTypeConversionError),
+                }
+            }
             _ => Err(MetaError::UnknownBqlTypeConversionError),
         }
     }
 
-    fn _decimal_type(decimal_item: &[u8]) -> MetaResult<Self> {
-        fn parse_num(bytes: &[u8]) -> MetaResult<u8> {
-            str::from_utf8(bytes)
-                .map_err(|_e| MetaError::UnknownBqlTypeConversionError)?
-                .trim()
-                .parse::<u8>()
-                .map_err(|_e| MetaError::UnknownBqlTypeConversionError)
+    fn _parse_num(bytes: &[u8]) -> MetaResult<u8> {
+        fn trim(bytes: &[u8]) -> &[u8] {
```

Comment:
Ok, I am not aware the need of trim? My mistake. How about moving trim related to base crate?  
There are [some functional trim reference impls](https://docs.rs/ascii/1.0.0/src/ascii/ascii_str.rs.html#191-193), just FYI.